Student_BCOs/JuneBeshea_BCO_000550.json

```diff
@@ -0,0 +1,237 @@
+{
+    "bco_id": "http://biocomputeobject.org/BCO_000550",
+    "checksum": "274de4a0875b62abc9095a6669fb7b57d079dd557aa78fc84a595167cfccae25",
+    "bco_spec_version": "1.3.0",
+    "provenance_domain": {
+        "name": "Correlating miRNA and gene expression data in 34 cancer phenotypes",
+        "version": "1.0.0",
+        "license": "Creative Commons Attribution 4.0 International License",
+        "created": "2019-10-17T02:13:35.166914",
+        "modified": "2019-10-17T03:16:35.853294",
+        "contributors": [
+            {
+                "name": "June Beshea",
+                "orcid": "https://orcid.org/0000-0001-9101-8000",
+                "affiliation": "George Washington University",
+                "contribution": [
+                    "createdBy"
+                ],
+                "email": "jbeshea@gwu.edu"
+            },
+            {
+                "name": "Micah Madrid",
+                "orcid": "https://orcid.org/0000-0002-7527-022X",
+                "affiliation": "George Washington University",
+                "contribution": [
+                    "contributedBy"
+                ],
+                "email": "mmadrid@gwu.edu"
+            },
+            {
+                "name": "Ling Sheng Loh",
+                "orcid": "https://orcid.org/0000-0003-0981-7984",
+                "affiliation": "George Washington University ",
+                "contribution": [
+                    "contributedBy"
+                ],
+                "email": "lingsloh@gwmail.gwu.edu"
+            }
+        ]
+    },
+    "usability_domain": [
+        "This workflow will facilitate the integration of already provided with TCGA data of mRNAs, miRNAs and or proteins in cancer, from there it will calculate the correlations the miRNA and expression data. This will allow better understanding of miRNA roles in expression profiles in cancer. This example pipeline was created based on the work of Ahmed et al. doi: 10.1186/s13104-018-3160-9"
```

Comment:
After reading the publication, did you write this content yourself?

Reply:
I read the publication, I thought this was supposed to align with group members! If not I will erase and write a new one.